package.json

```diff
@@ -60,6 +62,7 @@
     "@testing-library/react": "13.4.0",
     "@trivago/prettier-plugin-sort-imports": "^4.0.0",
     "@types/bcryptjs": "2.4.2",
+    "@types/crypto-js": "^4.1.1",
```

Comment:
Make sure you lock the package versions.

Reply:
I have locked package version.